evap/contributor/templates/contributor_index.html

```diff
@@ -186,12 +188,12 @@
                                             </a>
                                         {% elif evaluation.state != 'published' and evaluation.is_single_result %}
                                             <div class="d-flex" data-toggle="tooltip" data-placement="left" title="{% trans 'You will receive an email when the results are published.' %}">
-                                                {% include 'distribution_bar_disabled.html' with icon="fas fa-hourglass-half" weight=evaluation.weight weight_sum=evaluation.course.evaluation_weight_sum %}
+                                                {% include 'distribution_with_grade_disabled.html' with icon="fas fa-hourglass-half" weight=evaluation.weight weight_sum=evaluation.course.evaluation_weight_sum %}
```

Comment:
why not "distribution_widget_with_grade_disabled"? Is there a reason the other two are called widget but this is not?